velox/exec/tests/CMakeLists.txt

```diff
@@ -55,7 +55,9 @@ add_executable(
   TreeOfLosersTest.cpp
   UnnestTest.cpp
   VectorHasherTest.cpp
-  WindowFunctionRegistryTest.cpp)
+  WindowFunctionRegistryTest.cpp
+  AssignUniqueIdTest.cpp
```

Comment:
This test is present towards the beginning of this list. Please remove this entry.